src/test/amount_tests.cpp

```diff
@@ -67,12 +72,14 @@ BOOST_AUTO_TEST_CASE(GetFeeTest)
     feeRate = CFeeRate(1000);
     altFeeRate = CFeeRate(feeRate);
     BOOST_CHECK_EQUAL(feeRate.GetFee(100), altFeeRate.GetFee(100));
+    BOOST_CHECK(!altFeeRate.IsZero());
 }
 
 BOOST_AUTO_TEST_CASE(CFeeRateConstructorTest)
 {
     // Test CFeeRate(CAmount fee_rate, size_t bytes) constructor
     // full constructor
+    BOOST_CHECK(CFeeRate(CAmount(0), 0).IsZero());
```

Comment:
in b427ce178592283b9211b9fbd19874c7349bfd7e:

nit: I would have stuck these `IsZero()` tests in one place below since they are testing the function and not the ctor.

Reply:
done